tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -511,6 +511,13 @@ test_i2c_wsen_hids: wsen_hids@53 {
 	odr = "1";
 };
 
+test_i2c_wsen_tids: wsen_tids@38 {
+	compatible = "we,wsen-tids";
+	reg = <0x38>;
+	int-gpios = <&test_gpio 0 0>;
+	odr = <25>;
+};
+
```

Comment:
Please move this to the bottom of the file and use the next incremental address. This is a build test that will never execute on hardware so the address doesn't need to match the datasheet.

Reply:
Thank you Maureen, I was wondering what this address is. There's still a lot I need to know.